webapp/src/components/github_milestone_selector/index.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {id as pluginId} from 'manifest';
+import {getMilestones} from '../../actions';
+
+import GithubMilestoneSelector from './github_milestone_selector.jsx';
+
+const mapDispatchToProps = (dispatch) => ({
+    actions: bindActionCreators({getMilestones}, dispatch),
+});
+
+const mapStateToProps = (state) => ({
+    milestones: state[`plugins-${pluginId}`].milestones,
```

Comment:
It feels weird we are using the general state for this, specially since we are not using it on other similar places.

Reply:
@larkox We don't do this in other places because we don't need this kind of a name-number conversion. For instance, for the label widget, the label name is both what the user sees as an option, and what we pass to the GitHub API. But for milestones, the GitHub API expects the milestone number, not its name. 

Please also see https://github.com/mattermost/mattermost-plugin-github/pull/343#discussion_r467165842